plugins/out_pgsql/pgsql.c

```diff
@@ -92,6 +92,9 @@ static int cb_pgsql_init(struct flb_output_instance *ins,
         ctx->db_table = flb_sds_create(FLB_PGSQL_TABLE);
     }
 
+    /* connection options */
```

Comment:
you're not checking the return of flb_output_get_property not being a NULL value, this needs to be fixed

Reply:
Its value is currently null, so I did not check again

Reply:
I think it is not required. if the configuration was not provided, it continues currently behaviour.